### PR TITLE
libwebsockets 4.3.1

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
```
  CMake Error at /opt/homebrew/lib/cmake/libwebsockets/libwebsockets-config.cmake:35 (include):
    include could not find requested file:
  
      LwsCheckRequirements
```
I think they might have shipped a broken `.cmake` file.

**Comment by carlocab** [comment]
No, I take that back:
```
❯ fd cmake
libwebsockets/4.3.1/lib/cmake
libwebsockets/4.3.1/lib/cmake/libwebsockets/LibwebsocketsTargets-release.cmake
libwebsockets/4.3.1/lib/cmake/libwebsockets/LibwebsocketsTargets.cmake
libwebsockets/4.3.1/lib/cmake/libwebsockets/LwsCheckRequirements.cmake
libwebsockets/4.3.1/lib/cmake/libwebsockets/libwebsockets-config-version.cmake
libwebsockets/4.3.1/lib/cmake/libwebsockets/libwebsockets-config.cmake
```

**Review by SMillerDev — APPROVED** [review]
No idea how this could fix things but sure!

File: Formula/ttyd.rb [hunk 1]
```diff
@@ -27,10 +27,12 @@ class Ttyd < Formula
   uses_from_macos "vim" # needed for xxd
 
   def install
-    system "cmake", ".",
-                    *std_cmake_args,
-                    "-DOPENSSL_ROOT_DIR=#{Formula["openssl@1.1"].opt_prefix}"
-    system "make", "install"
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DOPENSSL_ROOT_DIR=#{Formula["openssl@1.1"].opt_prefix}",
+                    "-Dlibwebsockets_DIR=#{Formula["libwebsockets"].opt_lib/"cmake/libwebsockets"}",
```
Comment (carlocab) [inline Formula/ttyd.rb:32]: @SMillerDev This is the fix. Everything else is just refactoring

**Pull request closed without merging**